packages/salesforcedx-vscode-soql/test/vscode-integration/queryDataView/queryDataFileService.test.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { QUERY_RESULTS_DIR_PATH } from '../../../src/constants';
+import {
+  CsvDataProvider,
+  JsonDataProvider
+} from '../../../src/queryDataView/dataProviders';
+import { FileFormat } from '../../../src/queryDataView/queryDataFileService';
+import { mockQueryData, TestFileService } from '../testUtilities';
+
+describe('Query Data File Service', () => {
+  const documentName = 'example.soql';
+  const workspacePath = vscode.workspace.workspaceFolders![0].uri.fsPath;
+  const testResultsDirPath = path.join(workspacePath, QUERY_RESULTS_DIR_PATH);
+
+  afterEach(() => {
+    // delete the query-results directory and its files.
+    // @ts-ignore
+    fs.rmdirSync(testResultsDirPath, { recursive: true });
```

Comment:
Does this need to be ts-ignored?

Reply:
Yeah for some reason it does, maybe the version of TS we run doesn't have the correct types for `fs`?

[fs.rmdirSync() has an options property](https://nodejs.org/api/fs.html#fs_fs_rmdirsync_path_options) but TS thinks it only take one argument. It worked as expected and I did't dig much deeper into why TS was yelling at me in the test.

Reply:
Gotcah. Might be because the @types/node version is out of sync with the target node version